van-gogh-second-page/jons/index-tickets.css

```diff
@@ -0,0 +1,125 @@
+@import url('https://fonts.googleapis.com/css2?family=Varela+Round&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Noto+Sans+JP:wght@100;400;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Josefin+Sans&display=swap');
+
+/* Remove default style for elements  */
+* {
+    margin: 0px;
+    padding: 0px;
+    font-family: 'Varela Round', sans-serif;
+}
+
+input, button, submit { border:none; background: none; } 
+
+a {
+    background-color: none;
+    text-decoration: none;
+    color: inherit;
+}
+
+/* Styles */
+body {
+    background-color:#FFCE01;
+}
+
+header {
+    box-sizing: border-box;
+    padding-top: 40px;
+    display: flex;
+    justify-content: flex-start;
+    align-items: flex-start;
+    background-color: #FFCE01;
+    margin: 0px 55px 10px 55px;
+
+}
+header nav {
+    display: flex;
+    font-weight: bold;
+    font-size: 15px;
+    margin: 0;
+    padding: 0;
+    flex-flow: row wrap;
+    justify-content: flex-start;
+}
+
+.tickets-wrapper {
+    margin: 50px 85px 0px 85px;
+}
+
+h1 {
+    font-size: 3.5rem;
+}
+
+.right {
```

Comment:
I think it shouldn't be called right cause it occupied the whole width)

Reply:
Changed the name to "rightside-div" instead.
<img width="529" alt="Skärmavbild 2021-04-29 kl  19 35 26" src="https://user-images.githubusercontent.com/60552030/116594930-0c491e00-a923-11eb-9e5a-12a4fd4297b5.png">